cmd/bootstrap/main.go

```diff
@@ -216,14 +225,31 @@ func getTopology(peers sets.String) (string, []string, error) {
 			return "", nil, errors.Wrap(err, "check replication status")
 		}
 
+		replicaHost, err := db.ReportHost()
+		if err != nil {
+			return "", nil, errors.Wrap(err, "get report_host")
+		}
+		if replicaHost == "" {
+			continue
+		}
+
+		if peers.Len() > 1 {
```

Comment:
why do we need to check the number of alive peers?

Reply:
@hors ^^

Reply:
@hors don't have the explanation why this line (`if peers.Len() >1 {`) is needed